docs/01-overview/02-main-areas/application-connectivity/ra-02-configuring-runtime.md

```diff
@@ -1,19 +1,15 @@
 ---
 title: Configuring the Runtime
```

Comment:
It's more like an operation or a technical reference. Should it be moved?

Reply:
I think I forgot to remove it, as the file was duplicated in Technical References. Thanks for catching that! :)